packages/core/package.json

```diff
@@ -142,5 +143,8 @@
     "webpackbar": "^4.0.0",
     "yml-loader": "^2.1.0",
     "yn": "^3.0.0"
+  },
+  "devDependencies": {
+    "worker-loader": "^2.0.0"
```

Comment:
this should be moved to `dependencies` with the other loaders (`devDependencies` are not installed when you install canon-core in a different repo)

Reply:
You are right.